drivers/counter/counter_ll_stm32_rtc.c

```diff
@@ -627,11 +627,7 @@ static const struct stm32_pclken rtc_clk[] = {
 	STM32_CLOCK_INFO(0, DT_DRV_INST(0)),
 	/* Use Kconfig to configure source clocks fields (Deprecated) */
 	/* Fortunately, values are consistent across enabled series */
-#ifdef CONFIG_COUNTER_RTC_STM32_CLOCK_LSE
-	{.bus = STM32_SRC_LSE, .enr = RTC_SEL(1)}
-#else
 	{.bus = STM32_SRC_LSI, .enr = RTC_SEL(2)}
```

Comment:
Please remove the whole `#if DT_INST_NUM_CLOCKS(0) == 1` section.

Reply:
removed